paintBrush.js

```diff
@@ -0,0 +1,44 @@
+
+// Select the canvas in the dom
+const canvas = document.querySelector('canvas')
+// Sets the canvas to 2D drawing
+const ctx = canvas.getContext('2d')
+
+// Set canvas height in JS to match css height.
+canvas.width = 1100;
+canvas.height = 600;
+
+function drawLine(event){
```

Comment:
can we group the functions together. then put the procedural code together too, at the top